news/1258-default-cast-b8.rst

```diff
@@ -0,0 +1,42 @@
+New features
+------------
+
+* H5T_NATIVE_B8 types will now be cast to ``numpy.uint8`` by default
```

Comment:
& don't forget to update the news entry now that it affects all bitfield types. :slightly_smiling_face: